### PR TITLE
fix: Update README.md

### PR DESCRIPTION
**Pull request by lfinchHob**
Update quick-start to:
1. Clone Git repository as the docker compose file is now split out into multiple files. A single wget is no longer sufficient.
2. Create the state directory. Relying on docker compose to create it doesn't allow the unprivileged container user to write to it.
3. Change ownership of state directory to uid: 999, gid: 999 (unprivileged container user)

Closes # 

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

**Comment by talboren** [comment]
@lfinchHob thank you so much for this contribution!

File: README.md [hunk 1]
```diff
@@ -187,7 +187,10 @@ To turn PostHog off, set the `DISABLE_POSTHOG=true` environment variable and rem
 #### Spinning up Keep with docker-compose
 The easiest way to start with Keep is to run it via docker-compose:
 ```shell
-wget -O docker-compose.yml https://raw.githubusercontent.com/keephq/keep/main/docker-compose.yml
+git clone https://github.com/keephq/keep.git
```
Comment (talboren) [inline README.md:190]: what do you think about the following, to spare the user from having to clone the entire repo and ease of use:

```suggestion
wget -O docker-compose.common.yml https://raw.githubusercontent.com/keephq/keep/main/docker-compose.yml && wget -O docker-compose.yml https://raw.githubusercontent.com/keephq/keep/main/docker-compose.yml && mkdir -d 0777 -p state
```

let me know

Comment (talboren) [reply README.md:190]: Another interesting way might be just downloading some bash script that does all of this for you? (./start-keep.sh) or something, wdyt?

**Comment by lfinchHob** [comment]
Yep, I’ll throw it together and send a PR. Sent from my iPhoneOn 31 Dec 2023, at 18:32, talboren ***@***.***> wrote:﻿
@talboren commented on this pull request.



In README.md:
> @@ -187,7 +187,10 @@ To turn PostHog off, set the `DISABLE_POSTHOG=true` environment variable and rem
 #### Spinning up Keep with docker-compose
 The easiest way to start with Keep is to run it via docker-compose:
 ```shell
-wget -O docker-compose.yml https://raw.githubusercontent.com/keephq/keep/main/docker-compose.yml
+git clone https://github.com/keephq/keep.git

Another interesting way might be just downloading some bash script that does all of this for you? (./start-keep.sh) or something, wdyt?

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Pull request closed without merging**